broker/utils.go

```diff
@@ -422,17 +423,54 @@ func updateDomainMetadata(metadata *ContextMetadata, ctxElement *ContextElement)
 
 // NGSI-LD starts here.
 
+var ExpandOnce sync.Once
+var CompactOnce sync.Once
+var ldE *ld.RFC7324CachingDocumentLoader
+var ldC *ld.RFC7324CachingDocumentLoader
+
```

Comment:
mention the detail of the method before it.

Reply:
resolved